src/style.css

```diff
@@ -0,0 +1,68 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+* {
+  box-sizing: border-box;
+
+}
+
+.overall {
+  position: relative;
+  margin: 20px 0;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  bottom: 26px;
+
+}
+
+.iconsmall {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+  bottom: 16px;
+
+}
+
+.search {
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 62px;
+  font-family: inherit;
+  font-size: 16px;
+}
+
+.searchsmall {
+  font-size: 14px;
+  height: 42px;
+  padding-left: 33px;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff, 0%, #f6f6f7, 100%);
+  outline: none;
```

Comment:
it's better to include 'outline: none' in '.search' selector, not in the focused one.